src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java

```diff
@@ -132,6 +135,8 @@ public void testNoFlagsSupportedOS() {
         Assert.assertEquals(WorkloadInventoryReportModel.COMPLEXITY_EASY,workloadInventoryReportModel.getComplexity());
         // Workloads
         Assert.assertTrue(workloadInventoryReportModel.getSsaEnabled());
+
+        Assert.assertEquals(new SimpleDateFormat("yyyy-M-dd'T'hh:mm:ss.S").parse("2019-09-18T14:52:45.871Z"), workloadInventoryReportModel.getCreationDate());
```

Comment:
This check has to be moved after https://github.com/project-xavier/xavier-analytics/pull/32/files#diff-b3009355ef5fbe445e4bc348b7b04c14R125 because the field is coming from `BasicFields` agenda group

Reply:
I dont understand.

Reply:
The check
```
Assert.assertEquals(new SimpleDateFormat("yyyy-M-dd'T'hh:mm:ss.S").parse("2019-09-18T14:52:45.871Z"), workloadInventoryReportModel.getCreationDate());
```
has to be placed after check 
[`Assert.assertEquals("esx13.v2v.bos.redhat.com", workloadInventoryReportModel.getHost_name());`](https://github.com/project-xavier/xavier-analytics/blob/master/src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java#L122)
because, as you can understand from comment [`// BasicFields`](https://github.com/project-xavier/xavier-analytics/blob/master/src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java#L110), that is the block of checks for fields managed in the `BasicFields` agenda group.
Does it help?

Reply:
ohhhh, ok

Reply:
Probably using AssertJ and *.isEqualToIgnoringNullFields(expectedModel)* or .isEqualToComparingFieldByField(expectedModel)* would allow us to remove the individual lines of check.

I will update the tests putting that check line in that block, though.